Headers/state_chm_mod.F90

```diff
@@ -6616,6 +6642,7 @@ FUNCTION Ind_( name, flag ) RESULT( Indx )
 !   'N' or 'n' : Returns radionuclide species index
 !   'P' or 'p' : Returns photolysis species index
 !   'S' or 's' : Returns main species index (aka "ModelId")
+!   'T' or 't' : Returns transport tracer index
```

Comment:
Adding a new wildcard will also require an update in the Species Properties supplemental guide for RTD.

Reply:
This is info is now in the [Species Properties supplemental guide](https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/species-guide.html#transport-tracer-properties) (thanks to @yantosca).